apps/io-services-cms-webapp/src/utils/subscription.ts

```diff
@@ -24,6 +29,23 @@ type ErrorResponses =
 const isManageKey = (ownerSubscriptionId: NonEmptyString) =>
   ownerSubscriptionId.startsWith(MANAGE_APIKEY_PREFIX);
 
+const areUserAndOwnerEquals = (
```

Comment:
do we need really need a method just to call equal? :D

Reply:
009e4dc